admiral/pkg/clusters/admiralDatabaseClient.go

```diff
@@ -158,16 +178,16 @@ func NewDynamicConfigDatabaseClient(path string, dynamoClientInitFunc func(role
 	return &dynamicConfigClient, nil
 }
 
-func UpdateASyncAdmiralConfig(dbClient AdmiralDatabaseManager, syncTime int) {
+func UpdateASyncAdmiralConfig(rr *RemoteRegistry, syncTime int) {
```

Comment:
this func is running in a goroutine but we are not gracefully exiting it. Can we pass in the `ctx` and return gracefully when the program exits? 
https://github.com/istio-ecosystem/admiral/blob/3d33a9984778163ff99138970883d508e3e0205c/admiral/pkg/clusters/admiralReadWriteLeaseStateChecker.go#L67